test/integration_test.exs

```diff
@@ -0,0 +1,138 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.IntegrationTest do
+  use ExUnit.Case
+
+  alias Membrane.RTC.Engine.TimescaleDB.{
+    Cleaner,
+    Model,
+    Repo
+  }
+
+  alias Model.{
+    PeerMetrics,
+    TrackMetrics
+  }
+
+  setup do
+    Ecto.Adapters.SQL.Sandbox.checkout(Repo)
+    :ok
```

Comment:
This seems to assume `:manual` mode that you have to configure in `test_helper.exs`. Plus the example suggests matching the returned value
```suggestion
    :ok = Ecto.Adapters.SQL.Sandbox.checkout(Repo)
```